lib/hijack/http.js

```diff
@@ -21,6 +23,8 @@ if (Package['http']) {
       return originalCall.apply(this, arguments);
     }
 
+    mergeEnterWith({ isHttpAlreadyTracked: true });
```

Comment:
There are a number of bugs with Meteor.EnvironmentVariable in Meteor 3 at the moment. We could maybe wait on this until those get fix.

> Does the HTTP and fetch packages synchronously create the request? If so, we could use a variable to track it. 

Do you know if this would work?

Reply:
I could not find a way to use a variable to do that, there are simply too many layers between the calls, but using `run` and a new als store did the trick. The `Meteor.EnvironmentVariable` should work on Meteor 2 as an alternative I think.